### DISCUSSION TITLE
inifile on another path

### DISCUSSION THREAD

**Discussion by AT-jam** [jamulussoftware/jamulus#1704] [Technical Support & Help] [answered]
I tried to switch the path for inifile, but jamulus don't care about that and starts everytime with a blanco ini-configuration.
My commandline: "C:\Program Files\Jamulus\Jamulus.exe" -i "C:\Jamulus"
Whats wrong with this?

**Comment by hoffie** [comment, marked as answer]
"C:\Jamulus" looks like a path to a directory. Have you tried specifying a file path instead?

"C:\Program Files\Jamulus\Jamulus.exe" -i "C:\Jamulus\your.ini"

Also ensure that Jamulus has write permissions to this file.

**Reply by AT-jam** [reply]
Thats the solution, thanks!
